aws-opentelemetry-distro/src/amazon/opentelemetry/distro/exporter/otlp/aws/metrics/aws_cloudwatch_emf_exporter.py

```diff
@@ -0,0 +1,492 @@
+            for key, value in resource.attributes.items():
+                emf_log[f"otel.resource.{key}"] = str(value)
+
+        # Initialize collections for dimensions and metrics
+        metric_definitions = []
+        # Collect attributes from all records (they should be the same for all records in the group)
+        # Only collect once from the first record and apply to all records
+        all_attributes = (
+            metric_records[0].attributes
+            if metric_records and hasattr(metric_records[0], "attributes") and metric_records[0].attributes
+            else {}
+        )
+
+        # Process each metric record
+        for record in metric_records:
+
+            metric_name = self._get_metric_name(record)
+
+            # Skip processing if metric name is None or empty
+            if not metric_name:
+                continue
+
+            unit = self._get_unit(record.instrument)
+
+            # Create metric data dict
+            metric_data = {}
+            if unit:
+                metric_data["Unit"] = unit
+
+            # Process gauge metrics (only type supported in PR 1)
+            if not hasattr(record, "value"):
+                # Store value directly in emf_log
+                logger.debug("Skipping metric %s as it does not have valid metric value", metric_name)
+                continue
+
+            emf_log[metric_name] = record.value
+
+            # Add to metric definitions list
+            metric_definitions.append({"Name": metric_name, **metric_data})
+
+        # Get dimension names from collected attributes
+        dimension_names = self._get_dimension_names(all_attributes)
+
+        # Add attribute values to the root of the EMF log
+        for name, value in all_attributes.items():
+            emf_log[name] = str(value)
+
+        # Add the single dimension set to CloudWatch Metrics if we have dimensions and metrics
+        if dimension_names and metric_definitions:
+            emf_log["_aws"]["CloudWatchMetrics"].append(
+                {"Namespace": self.namespace, "Dimensions": [dimension_names], "Metrics": metric_definitions}
+            )
+
+        return emf_log
+
+    # pylint: disable=no-member
+    def _send_log_event(self, log_event: Dict[str, Any]):
+        """
+        Send a log event to CloudWatch Logs.
+
+        Basic implementation for PR 1 - sends individual events directly.
+
+        TODO: Batching event and follow CloudWatch Logs quato constraints - number of events & size limit per payload
+        """
+        try:
+            # Send the log event
+            response = self.logs_client.put_log_events(
+                logGroupName=self.log_group_name, logStreamName=self.log_stream_name, logEvents=[log_event]
+            )
+
+            logger.debug("Successfully sent log event")
+            return response
+
+        except ClientError as error:
+            logger.debug("Failed to send log event: %s", error)
```

Comment:
should this be `logger.error`? Otherwise customers won't know that the metric exports are failing unless they enable debug logging.

Reply:
nvm. I see the error being caught and logged in the broader `export` method.